libs/auth/package.json

```diff
@@ -25,10 +25,12 @@
     "@angular/common": "0.0.0-PLACEHOLDER",
     "@angular/core": "0.0.0-PLACEHOLDER",
     "@daffodil/core": "0.0.0-PLACEHOLDER",
+    "@daffodil/driver/magento": "0.0.0-PLACEHOLDER",
```

Comment:
We don't publish sub packages independently of the main package, so the dep here is simply `@daffodil/driver`.

Reply:
This dep is also an `optionalDependency`.

Reply:
fixed